### DISCUSSION TITLE
the max number words of a phrase doesnt work in chinese

### DISCUSSION THREAD

**Discussion by Tcould** [tadashi-aikawa/obsidian-various-complements-plugin#295] [Q&A]
Hello, after switching to Chinese, I set the max number words of a phrase to 6, but in actual use, I found that max number words of a phrase is only one or two, and English does not affect it, how to solve this？
![screenshots](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/127692738/254a7d3b-5e2f-4f99-affe-64d475c80104

**Comment by Tcould** [comment]
![screenshots](https://github.com/tadashi-aikawa/obsidian-various-complements-plugin/assets/127692738/a841d79b-19a8-45b7-94e5-4b67331a9587)

**Reply by tadashi-aikawa** [reply]
Hi, @Tcould :)

I'm sorry, I don't understand what the issue is. Could you please provide the steps to reproduce, the expected behavior, and the actual behavior?

For reference, the specifications for 'Max number of words as a phrase' can be found at the following link: [Max number of words as a phrase](https://tadashi-aikawa.github.io/docs-obsidian-various-complements-plugin/4.%20Options/4.1.%20Main/%E2%9A%99%EF%B8%8F%20Max%20number%20of%20words%20as%20a%20phrase/).
